packages/components/src/components/slider/slider.tsx

```diff
@@ -379,9 +387,9 @@ export class Slider {
                         role="slider"
                         id={this.sliderId + '-from'}
                         aria-valuemin={this.min}
-                        aria-valuenow={this.value}
+                        aria-valuenow={`${this.valueFrom} to ${this.valueTo}`}
                         aria-valuemax={this.max}
-                        aria-valuetext={`${this.value}`}
+                        aria-valuetext={`${this.valueFrom} to ${this.valueTo}`}
```

Comment:
```suggestion
                        aria-valuetext={this.getRangeAriaLabel()}
```

Reply:
I'm going to sound picky but I would name the prop `innerAriaValueText` and thus the helper function `getAriaValueText()`. Mainly because an `aria-label` is a different and specific kind of attribute; and also because it's not really a translation but rather the actual value you're setting in whatever language. (This wouldn't be consistent with other components, but we're not consistent a 100% yet either, and we'll definitely work on this in the near future.)

@felix-ico and @nowseemee do you think?

Reply:
updated it, was a mistake on my part.